.travis.yml

```diff
@@ -76,5 +76,4 @@ script:
   - curl -X GET http://127.0.0.1:5981/_all_dbs
   - i=$(curl -X GET http://127.0.0.1:5981/_all_dbs | jq length); if [ $i -ne 40 ]; then exit 1; fi
   - node -v
-  - (cd tests && codeceptjs run --steps --verbose)
-  - python -m unittest discover tests
+  - (cd tests && codeceptjs run --steps)
```

Comment:
@waryhermit do you really want to remove the python test line?
I would just comment the line ;)